pkg/proxy/healthcheck/healthcheck.go

```diff
@@ -215,7 +215,7 @@ func (h hcHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
 	} else {
 		resp.WriteHeader(http.StatusOK)
 	}
-	fmt.Fprintf(resp, strings.Trim(dedent.Dedent(fmt.Sprintf(`
+	fmt.Fprint(resp, strings.Trim(dedent.Dedent(fmt.Sprintf(`
```

Comment:
Looking at the godoc, I'm not sure this is what we want: https://golang.org/pkg/fmt/#Fprint

Reply:
The staticcheck warning is "printf-style function with dynamic format string and no further arguments should use print-style function instead (SA1006)". That looks correct, so in this case `Fprint` is equivalent to `Fprintf`.